cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -219,7 +219,7 @@
     require 'digest'
     node['bcpc']['hadoop']['oozie_hosts'].each do |oozie_host|
       next unless oozie_running?(float_host(oozie_host['hostname']))
-      node.set['bcpc']['hadoop']['oozie']['sharelib_checksum'] =
+      node.override['bcpc']['hadoop']['oozie']['sharelib_checksum'] =
```

Comment:
Why are we overriding in a recipe?

Recipe attr levels already have precedence

Reply:
we should keep everything in chef-bach at `default` so that internal wrappers will work.

Reply:
Corrected